src/commands/js_docker_prebuild_steps.yml

```diff
@@ -0,0 +1,11 @@
+parameters:
+  build_env:
+    type: enum
+    enum: [ 'beta', 'prod']
+    default: beta
+steps:
+  - run:
+      name: Copy env variable to .env file (to be used in Docker image by Sentry Webpack plugin)
```

Comment:
This is problematic. 
1. There is no build step involving webpack occurring in Docker. So webpack will not be run in Docker and the same for sentry-webpack-plugin. I could fine a trace of .env.sentry file in my-ricardo-spa in ricardo_modules (master), but for example, I couldn't find it in marketplace-spa. I'm wondering what we want to do with this file.
2. The printenv command will print the environment variables in the file .env.sentry. In my understanding, those environment variables are taken from contexts in circleci. In the dev context, I could find only one variable related to sentry : SENTRY_AUTH_TOKEN and in the prod context, I couldn't find anything. 
3. The only Sentry related variable I could find in my-ricardo-spa is SENTRY_PROJECT_URL. In marketplace-spa, I could find NEXT_PUBLIC_SENTRY_URL in sentry.client.config.js but I couldn't find this variable elsewhere, in isopod.yml, in .env, in the circleci context, etc.
4. If we have to share an environment variable in projects and docker images, I would recommend the preferred way: using isopod.yml and kubernetes secrets. Variables in circleci should be only live during quality-gate and/or build. Not runtime.

So, either I don't understand the whole thing, or if I understand well, this is completely useless.

Reply:
So basically => Don't rename. Just delete this useless thing.

Reply:
I fully agree on your comment. I blindly copy-pasted a job from marketplace-frontend-app. It looks like some outdated way of using sentry (3yrs old in mfa). 
I double checked the SENTRY_AUTH_TOKEN circleCI context var and it's not used. Since it exist only in dev context I think it's safe to remove it.

Reply:
Thank you for this.